### DISCUSSION TITLE
The chloroplast genome assembly resulted in only small fragments.

### DISCUSSION THREAD

**Discussion by WANGLU225** [Kinggerm/GetOrganelle#367] [Q&A]
Hello, Professor! I have been using GetOrganelle to assemble the chloroplast genome, but despite trying various parameters, I consistently obtain only small fragments. Could you please help me understand the possible reasons for this issue? The following two images show the assembly results and the parameter settings in Bandage.

![13ff5b4759f92272de139778c289aef](https://github.com/user-attachments/assets/34e97d09-2d4a-4fb2-bdeb-97bb8668a209)
![475c54deaaa68a08a4dc6d74206b11b](https://github.com/user-attachments/assets/d6f7e0d2-5dd1-4019-b08d-b325e800dd7a)
[get_org.log.txt](https://github.com/user-attachments/files/17856638/get_org.log.txt)

**Comment by Kinggerm** [comment]
1. Update GetOrganelle to the latest.
2. Increase input reads (--max-reads) when you have shallow-coverage results.
3. Bad word size choice, which forces the extension step to be less than one round. Try to increase it.
